### PR TITLE
Add Jellyfin Audio Player

### PR DESCRIPTION
**Pull request by luke2m**
This PR also fixes a small grammar error on the Clients page.

### CODE REVIEW

File: general/clients/index.md [hunk 1]
```diff
@@ -281,6 +292,17 @@ A third party app with direct play and HDR support. Available on iOS and Apple T
...
+
+A third party standalone music streaming app. Includes full support for background audio and casting.
+
+**Status** ✅ In Development, 3rd-Party
+
+**Links:**
+
+- [GitHub](https://github.com/leinelissen/jellyfin-audio-player)
+- [TestFlight](https://testflight.apple.com/join/cf2AMDpx)
+
```
Comment (dkanada) [inline general/clients/index.md:305]: Even if your client supports multiple platforms, you probably shouldn't copy the information in more than one place. The multiplatform section below might work, or adding a small section in the description with available platforms.

File: general/clients/index.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ Clients connect your devices to your Jellyfin server and let you view your conte
 > [!NOTE]
 > If you are interested in helping out, please see our [contribution guide](xref:contrib-index) and feel free to contact us for more information!
 >
-> They aren't on this page, some of them can be found at the [jellyfin-archive organization on GitHub](https://github.com/jellyfin-archive).
+> If it isn't on this page, some clients can be found at the [jellyfin-archive organization on GitHub](https://github.com/jellyfin-archive).
```
Comment (dkanada) [inline general/clients/index.md:13]: The subject here is `clients` which would require the use of `if they aren't` at the beginning of the sentence to indicate plurality.

File: general/clients/index.md (hunk 1 above)
Comment (luke2m) [reply general/clients/index.md]: Fixed.

File: general/clients/index.md (hunk 2 above)
Comment (luke2m) [reply general/clients/index.md]: Fixed.

**Review by dkanada — APPROVED** [review]

**Pull request merged**